pyvrp/PenaltyManager.py

```diff
@@ -116,14 +116,14 @@ class PenaltyManager:
         MAX_PENALTY]``.
     """
 
-    MIN_PENALTY = 1
-    MAX_PENALTY = 100_000
-    FEAS_TOL = 0.05
+    MIN_PENALTY: float = 0.1
```

Comment:
With `MIN_PENALTY = 0.1`, we can get situations where the excess load is non-zero but the resulting load penalty is zero. I'm not sure if that's desired and whether this change is needed.

Reply:
This is the default value in HGS-CVRP. Let's try if it works by benchmarking.